bool has_survived
float64 survival_time
float64 fitness